internal/codegen/stencil.go

```diff
@@ -177,7 +177,14 @@ func (s *Stencil) getTemplates(ctx context.Context, log logrus.FieldLogger) ([]*
 		}
 
 		log.Debugf("Discovering templates from module %q", m.Name)
-		err = util.Walk(fs, "", func(path string, inf os.FileInfo, err error) error {
+		baseDir := ""
```

Comment:
```suggestion
		var baseDir string
```

Reply:
don't tell me how to LIVE MY LIFE

Reply:
https://go-proverbs.github.io/

5th one down